qa/mixed-cluster/build.gradle

```diff
@@ -27,6 +27,24 @@ restResources {
   }
 }
 
+def excludeList = []
+
+// These tests check setting validations in the desired_node API.
+// Validation (and associated tests) are supposed to be skipped/have
+// different behaviour for versions before and after 8.10 but mixed
+// cluster tests may not respect that.
+// The qa cluster consists of 4 nodes. Two nodes are on old version and the
+// other two nodes on the current version. The node selector skips
+// the nodes on current version. The rest client then round robins
+// between the two nodes on old version. In order to unmute this,
+// we need a different node selector, that always consistently
+// selects the same node.
```

Comment:
We've actually tested this, it doesn't seem like to be the case.. Have you tried version skips for these tests and you still see them running in bwc tests?

Reply:
Oh nevermind, this is to exclude the tests entirely.

Reply:
Yes exactly. In mixed clusters you start with a version and then upgrade it to current; you may talk to nodes on different versions, non predictably, and end up with different behaviour based on which node you are talking to.
Also, the version skip is already there and works well in non-mixed scenarios, but for these tests it seems to apply to the "start" (older) version, so once upgraded you have "worked around" the skip.
I think I've see in other tests that the version is read from the node(s) and a suitable node is selected, but it does not seem like this case.